### DISCUSSION TITLE
Modules

### DISCUSSION THREAD

**Discussion by brucebailey** [MasaCMS/MasaCMS#107] [Q&A]
Just looking around for any fun or constructive modules that anyone would like to share.

**Comment by guustnieuwenhuis** [comment]
Hi @brucebailey,
We've shared a few on [Forgebox](https://www.forgebox.io/?search=masa+CMS).
We're currently working on new modules for the upcoming release.

**Comment by guustnieuwenhuis** [comment]
Hi @brucebailey, check out the 7.4 alpha release with new modules!

**Reply by brucebailey** [reply]
@guustnieuwenhuis thank you! I have it installed and it is pretty awesome! Thank you
